sources/VT100Output.h

```diff
@@ -13,6 +14,8 @@ typedef enum {
     MOUSE_BUTTON_SCROLLRIGHT = 7,// scroll right
     MOUSE_BUTTON_BACKWARD = 8,   // backward (4th button)
     MOUSE_BUTTON_FORWARD = 9,    // forward (5th button)
+    MOUSE_BUTTON_10,             // extra button 1
+    MOUSE_BUTTON_11,             // extra button 2
```

Comment:
Please add explicit values (` = 10`, ` = 11`) for these. It's a signal to future maintainers that the value of the enum is important and shouldn't be changed.